sycl/doc/extensions/experimental/sycl_ext_adaptivecpp_enqueue_custom_operation.asciidoc

```diff
@@ -0,0 +1,88 @@
+# sycl_ext_oneapi_enqueue_custom_operation
```

Comment:
Please rename this file to match the name of the extension.  Also reformat the content to follow our specification [template](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc).

Reply:
Thanks, have updated to use the ext template.